cmd/ketch/configuration/configuration.go

```diff
@@ -87,3 +104,43 @@ func (cfg *Configuration) DynamicClient() dynamic.Interface {
 	}
 	return i
 }
+
+// GetKetchConfigObject returns the unmarshalled contents of the config.toml file
+func (cfg *Configuration) GetKetchConfigObject() KetchConfig {
```

Comment:
Why do we need to make exactly same function at two different places? Can we make use of one?
https://github.com/shipa-corp/ketch/pull/86/files#diff-83f8b31f7d8a2c45f2588716bbad498d304e0b9fa479c4a274e572a82513404aR58

Reply:
The function that you linked is part of the mock. If it is not defined in the mock, it will not satisfy the interfaces it needs to and it will cause test failures.